tools - man pages update

kamalesh-babulal opened the pull request:
This patch series updates the man pages of the libcgroup tools
with `-b` option, which ignores the default systemd delegated hierarchy
path and constructs the path of the control groups relative to the
cgroup root hierarchy and also updates `cgget` tools ` -m` option,
which prints the current cgroup setup mode.

drakenclimber commented on doc/man/cgget.1:
```diff
@@ -55,6 +55,11 @@ This option can be used multiple times.
 print only values, not parameter names.
 
+.TP
+.B -m
```
Should this be after `-h` and before `-n`?  It appears the list was alphabetical before this change.

kamalesh-babulal replied:
@drakenclimber Thank you so much for the review comments. Agreed on all the comments, will re-work and push the v2.

drakenclimber commented on doc/man/cgget.1:
```diff
@@ -27,6 +27,12 @@ This parameter can be used multiple times.
 
+.TP
+.B -b
+Ignores the default systemd delegated hierarchy path and
```
super nitpick - the other options _didn't_ capitalize the first letter.  Should we do the same here?

drakenclimber commented on doc/man/cgxget.1:
```diff
@@ -43,6 +43,11 @@ If the system is running in cgroup v1 mode, libcgroup will convert the data as n
 
+.TP
+.B -b
+Ignores the default systemd delegated hierarchy path and constructs the path of the control groups
```
Same capitalization question.  Ironically, it looks like `cgdelete.1` capitalized every entry.  Weird.

drakenclimber commented on doc/man/cgset.1:
```diff
@@ -19,6 +19,12 @@ Set the parameters of input cgroups.
 
+.TP
+.B -b
+Ignores the default systemd delegated hierarchy path and
```
Same capitalization question. Ironically, it looks like `cgdelete.1` capitalized every entry. Weird.

drakenclimber commented on doc/man/cgxset.1:
```diff
@@ -36,6 +36,11 @@ data in/out of \fBcgxset\fR is in cgroup v2 format.
 
+.TP
+.B -b
+Ignores the default systemd delegated hierarchy path and constructs the path of the control groups
```
Same capitalization question

drakenclimber commented on doc/man/cgcreate.1:
```diff
@@ -20,6 +20,12 @@ rest of the defined control group’s files. These users are
 
+.TP
+.B -b
+Ignores the default systemd delegated hierarchy path and
```
Same capitalization question

kamalesh-babulal commented:
**V2 Changes**
- Addressed the review comments from @drakenclimber
   - Moved `-m` option in `cgget` to match the alphabetical order of options
   - Replaced the `-b` option's description from first letter Captial to lower case.

drakenclimber commented:
Pushed to main and release-3.0.  Thanks, @kamalesh-babulal
